stl/inc/execution

```diff
@@ -75,42 +75,71 @@ constexpr size_t _Still_active                = static_cast<size_t>(-1);
 
 // EXECUTION POLICIES
 namespace execution {
-    class sequenced_policy { // request for sequential execution with termination
+    class sequenced_policy {

```

Comment:
That would be a breaking change to forward declarations so I don't believe so. I think they are depicted as `class` in the spec so that's what we used. (I know we normally don't care about forward decls but I see no reason to go out of our way to break them here)

Reply:
I could have swore I commented elsewhere; given that they're depicted as class in the standard and c1xx' name mangling bug with that I prefer to leave them as class.